fe/fe-core/src/main/java/com/starrocks/server/MetadataMgr.java

```diff
@@ -342,7 +342,9 @@ public Statistics getTableStatisticsFromInternalStatistics(Table table, Map<Colu
             ConnectorTableColumnStats connectorTableColumnStats = columnStatisticList.get(i);
             if (connectorTableColumnStats != null) {
                 statistics.addColumnStatistic(columnRef, connectorTableColumnStats.getColumnStatistic());
-                statistics.setOutputRowCount(connectorTableColumnStats.getRowCount());
+                if (!connectorTableColumnStats.isUnknown()) {
+                    statistics.setOutputRowCount(connectorTableColumnStats.getRowCount());
```

Comment:
Is it possible that row count of statistics has been set repeatedly?

Reply:
Yes,but they are same